rero_mef/marctojson/do_gnd_agent.py

```diff
@@ -45,17 +54,41 @@ def __init__(self, marc, logger=None, verbose=False, transform=True):
         if transform:
             self._transform()
 
+    def get_type(self):
+        """Get type of record.
+
+        Entitäten der GND (Satztypen) 075 $b TYPE $2 gndgen
+        - b Körperschaft
+        - f Konferenz
+        - g Geografikum
+        - n Person (nicht individualisiert)
+        - p Person (individualisiert)
+        - s Sachbegriff
+        - u Werk
+        """
+        for field_075 in self.marc.get_fields('075') or []:
+            if field_075['2'] == 'gndgen':
+                return RECORD_TYPES.get(field_075['b'])
```

Comment:
What's happens with MARCXML python library if subfield '2' or 'b' isn't present ? Is a `KeyError` exception is raised ? If yes, this exception should be catched to return default/None value

Reply:
No exception here:
https://github.com/edsu/pymarc/blob/master/pymarc/field.py